components/sup/src/manager/service/supervisor.rs

```diff
@@ -64,64 +61,25 @@ impl fmt::Display for ProcessState {
     }
 }
 
-/// Additional params used to start the Supervisor.
-/// These params are outside the scope of what is in
-/// Supervisor.package.ident, and aren't runtime params that are stored
-/// in the top-level Supervisor struct (such as PID etc)
-#[derive(Debug, Deserialize, Serialize)]
-pub struct RuntimeConfig {
-    pub svc_user: String,
-    pub svc_group: String,
-    pub env_vars: HashMap<String, String>,
-}
-
-impl RuntimeConfig {
-    pub fn new(svc_user: String,
-               svc_group: String,
-               env_vars: HashMap<String, String>)
-               -> RuntimeConfig {
-        RuntimeConfig {
-            svc_user: svc_user,
-            svc_group: svc_group,
-            env_vars: env_vars,
-        }
-    }
-}
-
-impl Default for RuntimeConfig {
-    fn default() -> RuntimeConfig {
-        RuntimeConfig {
-            svc_user: util::users::DEFAULT_USER.to_string(),
-            svc_group: util::users::DEFAULT_GROUP.to_string(),
-            env_vars: HashMap::new(),
-        }
-    }
-}
-
 #[derive(Debug)]
 pub struct Supervisor {
     pub child: Option<HabChild>,
-    pub package: Arc<RwLock<PackageInstall>>,
     pub preamble: String,
     pub state: ProcessState,
     pub state_entered: Timespec,
     pub has_started: bool,
-    pub runtime_config: RuntimeConfig,
+    pid: Option<PathBuf>,
```

Comment:
We use this in `create_pidfile` to record what the path is before we delete it in `cleanup_pidfile`. After we do that, we set it to `None`